test/parallel/test-runner-run.mjs

```diff
@@ -551,3 +551,9 @@ describe('forceExit', () => {
     });
   });
 });
+
```

Comment:
Does this file still generate a warning when run? If not, we should add an assertion so it doesn't regress.

Reply:
Yes. The warning should be fixed by  https://github.com/nodejs/node/pull/53878

Reply:
Yeah, the warning still occurs. I don't think it can be avoided due to the number of tests running in parallel.